traits/ctraits.c

```diff
@@ -4094,87 +4094,18 @@ _trait_comparison_mode ( trait_object * trait, PyObject * args ) {
     if ( !PyArg_ParseTuple( args, "i", &comparison_mode ) )
         return NULL;
 
-    trait->flags &= (~(TRAIT_NO_VALUE_TEST | TRAIT_OBJECT_IDENTITY));
+    trait->flags &= (~(TRAIT_NO_VALUE_TEST | TRAIT_OBJECT_ID_TEST));
     switch ( comparison_mode ) {
         case 0:  trait->flags |= TRAIT_NO_VALUE_TEST;
                  break;
-        case 1:  trait->flags |= TRAIT_OBJECT_IDENTITY;
+        case 1:  trait->flags |= TRAIT_OBJECT_ID_TEST;
         default: break;
     }
 
     Py_INCREF( Py_None );
     return Py_None;
 }
 
-
-/*-----------------------------------------------------------------------------
-|  Sets the value of the 'setattr_original_value' flag of a CTrait instance:
-+----------------------------------------------------------------------------*/
-
-static PyObject *
```

Comment:
These removals somehow came out of resolving the conflicts. I'm not sure how, since both master and the #672 branch both contain these functions. Nevertheless, it's true that they're now unused, so can be removed.

Reply:
Ah, it's because c8fa0c9f0de6b1a02bf885ed06a41379c7c6d64d (the merge from master in #672) re-introduced the functions, presumably accidentally.